### DISCUSSION TITLE
Can you make a flexbox scrollable?

### DISCUSSION THREAD

**Discussion by mavam** [ArthurSonzogni/FTXUI#542] [Q&A] [answered]
I've tried making a flexbox scrollable, without much success. I tried modifying the flexbox gallery example as follows:

```diff
@@ -121,7 +121,7 @@ int main(int argc, const char* argv[]) {
     if (!group_yflex_grow)
       group = vbox(group, filler());
 
-    group = group | flex;
+    group = group | flex | vscroll_indicator | frame;
     return group;
   });
```

Is my approach wrong or is this not supported?

**Comment by mavam** [comment]
@ArthurSonzogni is this perhaps the same bug as in #546?

**Reply by ArthurSonzogni** [reply]
What does "group" contains?

One component must render itself with the `focus` decorator when focused, so that the `frame` will scroll the innerview to show it.

**Reply by mavam** [reply]
Ah, right. I think the example does not have focusable components, which is why just trying to make the outer renderer scrollable isn't working.

**Comment by mavam** [comment, marked as answer]
To answer my own question: the issue here is that simply adding a `vscroll_indicator` doesn't suffice.  The group must contain focusable elements, otherwise scrolling is not an option. This can be achieved by using an appropriate renderer or putting components into a container.
